modules/alloydb/README.md

```diff
@@ -0,0 +1,191 @@
+# AlloyDB module
+
+This module manages the creation of an AlloyDB cluster and instance with potential read replicas and an advanced cross region replication setup for disaster recovery scenarios. 
+It can also create an initial set of users via the `users` parameters.
+
+Note that this module assumes that some options are the same for both the primary instance and the secondary one in case of cross regional replication configuration.
+
+> [!WARNING]
+> If you use the `users` field, you terraform state will contain each user's password in plain text.
+
+<!-- TOC -->
+* [AlloyDB module](#alloydb-module)
+  * [Examples](#examples)
+    * [Simple example](#simple-example)
+    * [Cross region replication](#cross-region-replication)
+    * [Custom flags and users definition](#custom-flags-and-users-definition)
+    * [CMEK encryption](#cmek-encryption)
+  * [Variables](#variables)
+  * [Outputs](#outputs)
+<!-- TOC -->
+
+## Examples
+### Simple example
+
+This example shows how to setup a project, VPC and AlloyDB cluster and instance.
+
+```hcl
+module "project" {
```

Comment:
I think you can remove project and vpc. Look at other modules

Reply:
I don't think the default e2e project and vpc have the required configs (services, PSA, etc).

IMO a full example is nice, especially as a first example.